script/utils/generateApiSeeds/tableRecords.ts

```diff
@@ -89,22 +84,22 @@ export default class TableRecords {
 
       // one offering for active case load ID should have at least 100 referrals
       // to allow us to fully test features like pagination
-      const firstOfferingForActiveCaseLoadId = this.offeringsForActiveCaseLoadId()[0]
+      const firstReferableOfferingForActiveCaseLoadId = this.referableOfferingRecordsForActiveCaseLoadId()[0]
       while (records.length < 100) {
         records.push(
           this.referralRecord(
-            firstOfferingForActiveCaseLoadId.id,
+            firstReferableOfferingForActiveCaseLoadId.id,
             faker.helpers.arrayElement(prisoners).prisonerNumber,
           ),
         )
       }
 
       // some other offerings for active case load ID should have referrals
-      const nonFirstOfferingsForActiveCaseLoadId = this.offeringsForActiveCaseLoadId().slice(1)
+      const nonFirstReferableOfferingsForActiveCaseLoadId = this.referableOfferingRecordsForActiveCaseLoadId().slice(1)
```

Comment:
Not a huge problem but wonder if there's a better word/phrase than`nonFirst`? `subsequent`?

Reply:
That's way better!